packages/homepage/src/templates/docs.js

```diff
@@ -211,17 +211,20 @@ const Description = styled.p`
 // eslint-disable-next-line
 export default class Docs extends React.Component {
```

Comment:
Can you make this a functional component? Maybe after this one is merged

Reply:
Will do in a follow-up PR 🙂